trillian/migrillian/election/etcd/election.go

```diff
@@ -81,22 +66,16 @@ func (e *Election) Await(ctx context.Context) (context.Context, error) {
 			return nil, errMasterUnconfirmed
```

Comment:
Probably can return a closed context as well, and remove the err global variable.

Reply:
Done.